libs/SalesforceNetwork/SalesforceNetwork/SalesforceNetwork/Classes/Network/Action/CSFSalesforceAction.m

```diff
@@ -209,6 +222,18 @@ - (void)observeValueForKeyPath:(NSString *)keyPath ofObject:(id)object change:(N
     if (context == kObservingKey) {
         [self willChangeValueForKey:@"ready"];
         [self didChangeValueForKey:@"ready"];
+        if ([self requiresAuthentication] && (self.enqueuedNetwork.account == object)) {
```

Comment:
Pretty much the same as before, except it has a new home, and the extra ```[self requiresAuthentication]``` conditional.